framework/source/class/qx/util/DynamicScriptLoader.js

```diff
@@ -177,7 +180,10 @@ qx.Class.define("qx.util.DynamicScriptLoader", {
       dynLoader = cl.__IN_PROGRESS[script];
       if (dynLoader){
 
-          id1 = dynLoader.addListener('loaded',function(e){
+          id1 = dynLoader.addListener('loaded',function (e) {
+            if (this.$$disposed) {
```

Comment:
$$disposed is undocumented and should really be considered private. However, there is a public method for accessing whether the object is disposed. Let's change these to:
```
if (this.isDisposed()) {
```